web/src/stream_ui_updates.js

```diff
@@ -252,7 +252,13 @@ export function enable_or_disable_permission_settings_in_edit_panel(sub) {
 
     const $general_settings_container = $stream_settings.find($("#stream_permission_settings"));
     $general_settings_container
-        .find("input, select, button")
+        .find("input, button")
```

Comment:
I don't think there is any button in both the sections.

Reply:
Ohh, it can be due to disable the save-discard buttons if they are visible during live-update.

Reply:
Though, I think we should just hide the save-discard buttons in that case. I think we can do this as a separate PR, probably with #30466, and I will take care of it.

Reply:
"Who can unsubscribe others..." widget in advanced configuration is also a button.

Reply:
Ohh! Yes, I forgot that.

Reply:
Removed the selector. The button is hidden so we don't need to disable it.

Reply:
I don't think the CSS was originally added for disabling buttons, but since we are not changing any behavior for now, might be a fine way to handle the "admin unsubscribes user" case.